src/System.Windows.Extensions/src/System.Windows.Extensions.csproj

```diff
@@ -27,8 +27,8 @@
     <Compile Include="$(CommonPath)\Interop\Windows\CryptUI\Interop.CryptUIDlgCertificate.cs">
       <Link>Common\Interop\Windows\Crypt32\Interop.CryptUIDlgCertificate.cs</Link>
     </Compile>
-    <Compile Include="$(CommonPath)\Interop\Windows\User32\Interop.MessageBeep.cs">
-      <Link>Common\Interop\Windows\User32\Interop.MessageBeep.cs</Link>
+    <Compile Include="$(CommonPath)\Interop\Windows\user32\Interop.MessageBeep.cs">
+      <Link>Common\Interop\Windows\user32\Interop.MessageBeep.cs</Link>
```

Comment:
We should make these upper case.  If it's not on the file system, we should rename the directory as well.